test/parallel/test-assert-fail.js

```diff
@@ -23,7 +23,7 @@ assert.throws(
     code: 'ERR_ASSERTION',
     type: assert.AssertionError,
     operator: undefined,
-    actual: undefined,
+    actual: 'custom message',
```

Comment:
I remember now. Before this change `expectsError` only checked [`message`, `type`,`code`] now it compares `name` if provided. And for `AssertionError` also ['generatedMessage', 'actual', 'expected', 'operator']